src/main/java/uk/gov/hmcts/cmc/claimstore/config/JacksonConfiguration.java

```diff
@@ -25,4 +56,29 @@ public ObjectMapper objectMapper() {
             .disable(SerializationFeature.FAIL_ON_EMPTY_BEANS);
     }
 
+    @Bean
+    public ObjectMapper ccdObjectMapper() {
```

Comment:
Would be nice to this configuration in ccd-adapter module under `uk.gov.hmcts.cmc.ccd.config`